lib/directory-view.coffee

```diff
@@ -35,6 +35,7 @@ class DirectoryView extends HTMLElement
     @directoryName.textContent = @directory.name
     @directoryName.dataset.name = @directory.name
     @directoryName.dataset.path = @directory.path
+    @directoryName.draggable = true
```

Comment:
It would be great if it were the `.list-item` parent element that were draggable, rather than the name - currently, you have to have the cursor over the text, otherwise the item can't be dragged, and if you're dragging it to a folder, it has to be dropped on the folder's text.